draft annsa description (#9)
kamuda1/dissertation: theory.tex

\subsection{General Neural Network Architecture}

% Node/neruon, get this lingo straight here

> An ANN is a mathematical model that attempts to map an arbitrary function from $\mathbb{R}{^M}$ to $\mathbb{R}{^N}$, where $M$ and $N$ are positive integers. An ANN accomplishes this by mimicking biological neurons. One example of an ANN architecture is shown in Figure \ref{fig:Network}. This ANN has N neurons in input layer A, J neurons in layer B, and K neurons in output layer C. Layer B is called a hidden layer because it is not observed by in the input or output. Each neuron in adjacent layers are connected by weights, represented in Figure \ref{fig:Network} by arrows connecting neurons. 

Reviewer: Plenty of grammatical issues like this in the PR. Here's one in the first changed line... 

```suggestion
An ANN is a mathematical model that attempts to map an arbitrary function from $\mathbb{R}{^M}$ to $\mathbb{R}{^N}$, where $M$ and $N$ are positive integers. An ANN accomplishes this by mimicking biological neurons. One example of an ANN architecture is shown in Figure \ref{fig:Network}. This ANN has N neurons in input layer A, J neurons in layer B, and K neurons in output layer C. Layer B is called a hidden layer because it is not observed in the input or output. Neurons in adjacent layers are connected by weights, represented in Figure \ref{fig:Network} as arrows connecting the neurons. 
```
